ext/drreg/drreg.c

```diff
@@ -655,7 +655,6 @@ drreg_forward_analysis(void *drcontext, instr_t *start)
     for (reg = DR_REG_START_GPR; reg <= DR_REG_STOP_GPR; reg++) {
```

Comment:
> Resetting the spilled flag is not needed as it will be set in drreg_reserve_reg_internal().

This is incomplete logic: there are other paths like drreg_reserve_aflags().  Please change this to cite #3827 or something as it is not 100% clear what the consequences are of not resetting, at first glance.

Reply:
Done.